cli/config/configfile/file_windows.go

```diff
@@ -0,0 +1,5 @@
+package configfile
+
+func copyFilePermissions(src, dst string) {
+	// TODO implement for Windows
```

Comment:
Question: what would be the implementation for windows?

Reply:
Not sure how to copy permissions and/or ownership on Windows. Perhaps @simonferquel would know.

Also not sure if the problem that this was fixing happens on Windows as well

Reply:
Hmm, I am not sure about the correct "Windows" way to do that. But anyway I don't think it applies (when you create a file on Windows, by default you inherit ACLs from the parent directory).

Reply:
Thx! Perhaps I should update the comment and remove "todo"

Reply:
also renaming the function would be welcome, because on the windows implementation it does not do what it claims to do (in the general case it does not copy the permissions, which is fine in this specific case, but could be confusing in the future). Maybe renaming it to `replicateConfigFilePermissions` or something like this would be preferable.

Reply:
Don't think it's needed to include "configfile" in the name, as it's already in this package, and in itself, it's not specific to configfiles.

I'll have a look for changing the comment (and possibly name) on master